pom.xml

```diff
@@ -53,7 +53,7 @@
         <swagger.version>2.0.10</swagger.version>
         <version.jakartaee>9.0.0</version.jakartaee>
         <version.jetty-maven-plugin>11.0.8</version.jetty-maven-plugin>
-        <version.wildfly>25.0.1.Final</version.wildfly>
+        <version.wildfly>26.0.1.Final</version.wildfly>
```

Comment:
@jamezp I upgraded the wildfly version in `bootable jar` example, I compiled it and run with

```bash
➤ java -jar target/products-runner-bootable.jar
```

And I get the following error:

<img width="796" alt="image" src="https://user-images.githubusercontent.com/201907/160424392-5225e855-0489-4157-80cb-e2edf3878e26.png">

This seems caused by the upgrade of the wildfly plugin. Could you please teach me how to fix this?